x/config/tls_test.go

```diff
@@ -37,7 +37,7 @@ func TestTLS_SNI(t *testing.T) {
 	require.NoError(t, err)
 	cfg := tls.ClientConfig{ServerName: "host", CertificateName: "host"}
 	for _, option := range options {
-		option("host", 443, &cfg)
+		option("host", &cfg)
```

Comment:
FYI, this change was needed due to the update of the dependency on the main SDK module